dumper/lib-ext-hive-metastore/src/main/java/com/google/edwmigration/dumper/ext/hive/metastore/HiveMetastoreThriftClient.java

```diff
@@ -159,7 +167,7 @@ private TTransport createTransport() throws SaslException, TTransportException {
 
       Map<String, String> saslProperties = new HashMap<>();
       saslProperties.put(Sasl.SERVER_AUTH, "true");
-      saslProperties.put(Sasl.QOP, "auth-conf");
+      saslProperties.put(Sasl.QOP, saslQop);
```

Comment:
Current code implies that if nullable `kerberosUrl` is not null, the `saslQop` is also not-null. Consider adding a nullnes check  for `saslQop` too.

Reply:
I feel `saslQop` can never be null as it has a default value. Let me rather add `@NonNull` before `saslQop`